### ISSUE TITLE
Move DALI framework into local workspace

### ISSUE THREAD

**Issue by timofei7** [dali-lab/Lab-app#22]
https://github.com/dali-lab/lab-app-iOS/blob/f7b24515aa3bb0c3ee4ec28fc2280792adbd8ec1/Podfile#L13

https://github.com/dali-lab/lab-app-iOS/blob/f7b24515aa3bb0c3ee4ec28fc2280792adbd8ec1/Podfile#L41

**Comment by johnlev** [comment]
Yeah, it made it so I could work on it in tandem. I’m wondering if I should just shut down the cocoa pod since no one is using it and just make it a framework in the app

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by timofei7** [comment]
Ah yeah, that might be better since the app became a larger product rather than a set of smaller tools.  we ran into some swift 5 issues trying to get it running on our machines, might be some of the pods, will try a bit more later today.

**Comment by timofei7** [comment]
@johnlev lemme know if you're thinking of taking down the cocoapod when you have a chance

**Comment by timofei7** [comment]
paths fixed in https://github.com/dali-lab/lab-app-iOS/pull/23

**Comment by johnlev** [comment]
I might, but most likely I won’t have time. Maybe on the plane ride this Saturday. I’ll let you know by then if I can’t

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
